sdk/keyvault/keyvault-admin/src/lro/selectiveKeyRestore/operation.ts

```diff
@@ -108,30 +76,50 @@ export class KeyVaultSelectiveKeyRestorePollOperation extends KeyVaultAdminPollO
     }
 
     if (!state.isStarted) {
-      const selectiveRestoreOperation = await this.selectiveRestore(keyName, {
-        ...this.requestOptions,
-        restoreBlobDetails: {
+      // Are there any codegen helpers that can be used here?
+      const selectiveKeyRestoreOperation = this.client.selectiveKeyRestoreOperation(
+        keyName,
+        {
           folder: folderName,
           sasTokenParameters: {
             storageResourceUri: folderUri,
             token: sasToken,
             useManagedIdentity: sasToken === undefined,
           },
         },
-      });
-      this.mapState(selectiveRestoreOperation);
+        this.requestOptions,
+      );
+      await selectiveKeyRestoreOperation.poll();
```

Comment:
🤷 no idea what should be done here. Create a poller, poll once, get result?

Reply:
I think you can just `await selectiveKeyRestoreOperation`